spec/helper/functional-test.helper.js

```diff
@@ -348,14 +314,30 @@ class FunctionalTest {
                 }
             });
         }).then((response) => {
-            this.state.sessionId = response.body.sessionId;
+            var body;
+
```

Comment:
You've added great error checking elsewhere. Would you add it here as well or does that break tests?